tests/cupyx_tests/scipy_tests/sparse_tests/test_linalg.py

```diff
@@ -55,3 +55,22 @@ def test_ndarray(self, xp, sp):
         b = xp.array(self.b, dtype=self.dtype)
         x = sp.linalg.lsqr(A, b)
         return x[0]
+
+
+
+@testing.parameterize(*testing.product({
+    'ord': [None , -numpy.Inf, -2, -1, 0, 1, 2, 3, numpy.Inf],
+    'dtype': [numpy.float64, numpy.float64],
```

Comment:
```suggestion
    'dtype': [numpy.float32, numpy.float64],
```
and `numpy.complex64, numpy.complex128` maybe.

Reply:
changed tests working fine